### ISSUE TITLE
Profile restore to "By power Surplus" if selected.

### ISSUE THREAD

**Issue by chana1900** [montekarlos/homebridge-fronius-inverter-output-pin-switch#1]
I tested this on a Fronius Symo 10.0-3-M with a Fronius Smart Meter 63A and is working. It updates the Load Management profile to turn ON the relay when operated via Homebridge.

The issue I'm facing is that when it is restoring the original Load Management profile, it restore to the controlling option "by power production" rather than the "by power surplus"

Please see the profile I am using. It takes the "Feed in" into account when turning on the relay. 

It is great if this can be implemented.

<img width="821" alt="Fronius-pm-profile" src="https://github.com/montekarlos/homebridge-fronius-inverter-output-pin-switch/assets/91853853/0475bd31-7328-4672-9354-155b33004ea0">

**Comment by chana1900** [comment]
I fixed it by changing 
`pinConfig.mode = 'ppv';`
to 
`pinConfig.mode = 'pgrid';`
in _setPinPpvConfig_ in 'fronius.ts'

**Comment by montekarlos** [comment]
Yes this was a bit of a hack just for my usage. A better solution is to read the config in checkPinStateAtInverter (fronius-ouptut-pin-accessory.ts) and persistant so it can be restored - just like the on / off thresholds
